systers_portal/templates/meetup/meetup.html

```diff
@@ -0,0 +1,14 @@
+{% extends "meetup/base.html" %}
+
+{% block meetup_location_page_content %}
+  <h1 class="text-center">{{ meetup.title }}</h1>
+  <div class="box-container box-body">
+    {{ meetup.description|safe }} <br/>
+    {{ meetup.date }}
+    {{ meetup.time }} <br/>
+    {{ meetup.venue }}
+  </div>
+  <div class="box-container box-body">
+    <h3 class="text-center"> Discussion Area </h3>
+  </div>
+{% endblock %}
```

Comment:
We need to wok a bit on the template. Everything else looks fine.
If you want, we can go back to the mockup (I see you made one).

First off, we need to put the meetup metadata someplace where it's more visible and distinguishable, maybe at the top of the section? Specify exactly that this is the Time and the Venue, because otherwise it looks like a random timestamp (might be the time the meetup was created). Think ahead where is going to be the RSVP section, but don't mark it for now. Same for the discussion area, just remove the box, because if we don't manage to get to any of those, this stuff will stay there and will look unfinished.

The meetup title can't be h1, because it doesn't follow the hierarchy correctly. The meetup location title is h1. This semantic stuff is important for accessibility.